tests/test_mprotect.py

```diff
@@ -2,11 +2,13 @@
 
 import pwndbg
 import tests
+from pwndbg.gdblib import arch_mod
 
 MPROTECT_BINARY = tests.binaries.get("mprotect.out")
 
 
 def test_mprotect(start_binary):
+    arch_mod.update()
```

Comment:
Is this necessary at all? I think this should execute automatically when we start the binary/load an objfile iirc.

Reply:
Yes, it isn't called during tests